packages/common_client/lib/src/ld_common_client.dart

```diff
@@ -528,6 +576,12 @@ final class LDCommonClient {
     LDEvaluationDetail<LDValue> detail;
 
     if (evalResult != null && evalResult.flag != null) {
+      if (evalResult.flag?.prerequisites != null) {
```

Comment:
I think you could remove this check entirely. As it is covered by `?.forEach`

Reply:
Done